src/handlers/package.js

```diff
@@ -46,3 +74,32 @@ export const verifyPackagePaymentHandler = async (req, res) => {
         res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
     }
 }
+
+
+export const addScheduleHandler = async (req, res) => {
+    try {
+        const id = req.params.packageId;
+        const add = await addScheduleController(id, req.body)
```

Comment:
Rename variable add to code, and should be integer instead of boolean, so that more than two conditions can be checked.